wonnx-py/tests/test_specific_op.py

```diff
@@ -13,7 +13,7 @@
 
 backend_test = onnx.backend.test.BackendTest(DummyBackend, __name__)
 
-backend_test.include(f"test_{OP_TESTED}_[a-z,_]*")
```

Comment:
Not really related to implementing basic grouped convolutions (but fine with me)